requirements/requirements.txt

```diff
@@ -2,6 +2,7 @@ huggingface_hub>=0.24
 numba
 numpy>=1.22
 onnx>=1.7.0
+onnxruntime-gpu>=1.20.1
```

Comment:
Well we kind of kept onnxruntime optional - should not be in requirements.

Reply:
We need the onnx runtime. Moving it to the requirements_nlp.txt. This file is also used for building the container.